lib/flavors.dart

```diff
@@ -23,4 +23,22 @@ class F {
         return 'https://nimble-survey-web-staging.herokuapp.com';
     }
   }
+
+  static String get clientId {
+    switch (appFlavor) {
+      case Flavor.production:
+        return '4gg3bokkvPnMxWz7HHTdM_wf1RNg9k8iA6sZ2ZrA7EA';
+      default:
+        return 'z9iUamZLvRgtVVtRJ8UqItg2vmncGyEi30p1eWEddnA';
+    }
+  }
+
+  static String get clientSecret {
+    switch (appFlavor) {
+      case Flavor.production:
+        return 'y_GgV-GEjWd3VTzbZBS6tqEco0E68QuqHQv0QND2vKo';
+      default:
+        return '1vqRNMxq-Yx83A61GNjLb17qxCGKxHDb8EmB3MKdxqA';
+    }
```

Comment:
It's better to get these values from `.env` file to protect them, what do you think?
You can use this [lib](https://pub.dev/packages/flutter_config) to load these values from `.env`

Reply:
Yes, flutter_config is better in this case, please take a look at that @chornerman. You can check out my implementation here https://github.com/luongvo/flutter-survey/blob/develop/.env.sample